test/napi/lib/functions.js

```diff
@@ -114,4 +114,8 @@ describe('JsFunction', function() {
     assert.equal(addon.is_construct.call({}).wasConstructed, false);
     assert.equal((new addon.is_construct()).wasConstructed, true);
   });
+
+  it('should be able to call a function from an FnMut', function() {
```

Comment:
I tested manually that closures get dropped, but it would be nice to have an explicit test.